README.md

```diff
@@ -29,14 +29,14 @@ See our standards regarding:
 
 Find out about our processes:
 
-- [Find something to work on](process.md#finding-something-to-work-on)
-- [Propose features](process.md#proposing-features)
-- [Contributor ladder](process.md#contributor-ladder)
-- [Pull request reviews](process.md#reviews)
-- [Propose projects](process.md#proposing-projects)
+- [Find something to work on](./process/README.md#finding-something-to-work-on)
+- [Propose features](./process/README.md#proposing-features)
+- [Contributor ladder](./process/contributor-ladder.md)
+- [Pull request reviews](./process/README.md#reviews)
```

Comment:
I suggest also linking pull request process? like you did in process.README.md?

Reply:
done, thanks!